main.c

```diff
@@ -5,7 +5,7 @@ static char *compute_shader_paths[CS_LAST] = {
 	[CS_HADAMARD] = "shaders/hadamard.glsl",
 	[CS_LPF]      = "shaders/lpf.glsl",
 	[CS_MIN_MAX]  = "shaders/min_max.glsl",
-	[CS_UFORCES]  = "shaders/uforces.glsl",
+	[CS_UFORCES]  = "shaders/2d_hercules.glsl",
```

Comment:
This should go to `CS_HERCULES` which will need to be uncommented in `beamformer.h`.

Reply:
Updated